### ISSUE TITLE
Minet twitter followers and friends

### ISSUE THREAD

**Issue by ameliepelle** [medialab/minet#387]
Minet followers and friends command only returns user ids, but with API v2 we can have more metadata about the followers and following users of an account. Should we change the output of those commands for --api-v2 flag ?

**Comment by boogheta** [comment]
If that's possible within less API calls I guess we should!

**Comment by Yomguithereal** [comment]
Yes but there is the issue that you are likely to get a lot of user data duplicated (potentially bloating the output whereas it could also be lean) in the output so I still need we need to think about a way to enable/disable hydration with v2 and this poses a UX problem we need to discuss.
